server/config/constants.js

```diff
@@ -1,10 +1,7 @@
-const ROLE_ENUM = {
-  0: 'SUPER_ADMIN',
-  1: 'ADMIN',
-  2: 'USER',
-  SUPER_ADMIN: 0,
-  ADMIN: 1,
-  USER: 2,
+const ROLES_ENUM = {
+  SUPER_ADMIN: 'SUPER_ADMIN',
```

Comment:
I suggest to keep it related to numbers, So comparing between user's roles will be easier (game of numbers & not string comparing).

Reply:
Yes, I also thought but numbers are not readable and bit confused, so we have to make another enum on client end side.